test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +44,34 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+    
+
```

Comment:
Extra blank line here :P 

It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.